Library/Homebrew/service.rb

```diff
@@ -109,51 +109,46 @@ def run(command = nil, macos: nil, linux: nil)
 
     sig { params(path: T.nilable(T.any(String, Pathname))).returns(T.nilable(String)) }
     def working_dir(path = nil)
```

Comment:
```suggestion
sig { params(path: T.any(String, Pathname)).returns(T.nilable(String)) }
def working_dir(path = T.unsafe(nil))
```
@dduugg thoughts on this form?

Reply:
I'm fine with it, if you don't mind narrowing the existing contract. Lemme update the code, and we can see what breaks 😬